docs/eventing/getting-started.md

```diff
@@ -0,0 +1,416 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative workflow. You will learn how to create, send, and verify Knative events.
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](https://knative.dev/docs/install/index.html). 
+
+## Set up components
+
+Before you start to send Knative events, you need to create the components needed to move the events. In this guide, you will be creating components individually, but you can also create components by deploying a [single YAML file](https://raw.githubusercontent.com/akashrv/docs/qs/docs/eventing/samples/hello-world/quick-start.yaml).
+
+### Configure namespace for eventing
```

Comment:
Whats the bigger task/goal here? We say configuring a namespace, but what is the large "task" that the user should accomplish? I honestly dont know the answer here. But below, for example, it is mentioned that "creating a namespace also creates a 'broker'". We should follow up and find out if thats the primary reason, otherwise whats missing here is the "WHY". 

Ie. "Setting the default Eventing namespace to ...[explain more here to help user understand this concept]"

Use this as an example for all other individual "steps" and take advantage of that opportunity to "teach" about the "why create/config/blah...." a namespace/etc. So for example when or if you later need to "build on the concept of a Knative ....." (in this example the Knative Eventing namespace), then its easier and that base has been established.

Think of each step as a build block and take full advantage of the opportunity to explain in-depth, that related concept.